api/src/main/java/org/openmrs/util/UpgradeUtil.java

```diff
@@ -38,17 +38,19 @@ public static Integer getConceptIdForUnits(String units) {
 		String appDataDir = OpenmrsUtil.getApplicationDataDirectory();
 		Properties props = new Properties();
 		String conceptId = null;
-		try {
-			FileInputStream fis = new FileInputStream(appDataDir + System.getProperty("file.separator")
-			        + DatabaseUtil.ORDER_ENTRY_UPGRADE_SETTINGS_FILENAME);
+		String filePath = new StringBuilder(appDataDir)
```

Comment:
@JudeNiroshan dont extract this temp variable. just move the fis into the try-with resource like you did in the other cases.